WordPressAuthenticator/Authenticator/WordPressAuthenticator.swift

```diff
@@ -55,19 +59,24 @@ import WordPressUI
 
     /// Designated Initializer
     ///
-    private init(configuration: WordPressAuthenticatorConfiguration, style: WordPressAuthenticatorStyle) {
+    private init(configuration: WordPressAuthenticatorConfiguration,
+                 style: WordPressAuthenticatorStyle,
+                 text: WordPressAuthenticatorDisplayText) {
```

Comment:
Nipticky comment here!!

I'm not sure if `text` is the best property name. Although its uncountable, feels like a "single string" for a "single spot". How about....

- **displayStrings**
- **localizedStrings**

This one caught my eye, mostly, because UILabel's `.text` property!